templates/zerver/login.html

```diff
@@ -131,6 +131,12 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
                 {% endif %}
             </div>
         </div>
+
+        {% if realm_invite_required %}
+        <div class="contact-admin">
+            <p class="invite-hint">Don't have an account yet? Contact the organization administrators for an invitation.</p>
```

Comment:
This should be tagged for translation.  https://zulip.readthedocs.io/en/latest/translating/internationalization.html

Reply:
okay I will make this change